iquip/apps/scheduler.py

```diff
@@ -234,9 +286,31 @@ def __init__(self, name: str, parent: Optional[QObject] = None):
         """Extended."""
         super().__init__(name, parent=parent)
         self.schedulerFrame = SchedulerFrame()
+        self.thread = _ExperimentQueueFetcherThread(
+            self._snycExperimentQueue,
+            self
+        )
+        self.thread.start()
+
+    def _snycExperimentQueue(self,
+                             experimentList: List[SubmittedExperimentInfo],
+                             runningExperiment: Optional[SubmittedExperimentInfo] = None,
+        ):
+        """Displays the experiments fetched from the uploaded queue in the proxy server.
+
+        Args:
+            experimentList: The queue of pending experiments.
+            runningExperiment: The experiment running now. None if there is no experiements running.
+        """
+        self.schedulerFrame.model.experimentQueue = experimentList
+        self.schedulerFrame.model.dataChanged.emit(self.schedulerFrame.model.index(0), # refresh
+                                                   self.schedulerFrame.model.index(0),
+                                                   [Qt.EditRole])
+        self.schedulerFrame.model.experimentQueue.sort(key=lambda x: x.arginfo["priority"],
```

Comment:
The `sort()` seems to be called frequently with the same option. Is there a way to simplify it?